en/ros/ros1_via_ros2.md

```diff
@@ -72,6 +78,18 @@ As discussed in [ROS 2 User Guide > Sanity Check the Installation](../ros/ros2_c
 To use ROS (1) **and** ROS 2 (you need both for this!):
 
 1. [Setup your PX4 Ubuntu Linux development environment](../dev_setup/dev_env_linux_ubuntu.md) - the default instructions get the latest version of PX4 source and install all the needed tools.
+
+   :::note
+   The above instructions must be adjusted to get version v1.13 of PX4 source.
+   
+   ```sh
+   git clone -b <tag> https://github.com/PX4/PX4-Autopilot.git --recursive
```

Comment:
@beniaminopozzan What you have as `tag` here is actually a branch - https://github.com/PX4/PX4-Autopilot/branches/all?query=release

Releases are tagged as well, but don't have the `release/` prefix: https://github.com/PX4/PX4-Autopilot/releases?q=release&expanded=true

What is it that we recommend people use?

Reply:
Note, I THINK that the release branch will reflect latest-and-greatest, while the tag comes of that and reflects a specific point build. So we should suggest the branch "first".

Reply:
I was a bit unsure about how to phrase that. My understanding is that the tags are the stable versions while the `release/` branches are where relevant commits are backported from `main` in preparation of an update of the releases, right?

So as this piece of doc is just for the release `1.13.*` I just listed all tags and branches which are compatible to it. If I had to choose, I'll say that `v1.13.2` it currently the recommended one.